eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/configuration/batch/steps/CreateSubsnpAccessionsStepConfiguration.java

```diff
@@ -72,6 +74,9 @@ public Step createSubsnpAccessionStep(StepBuilderFactory stepBuilderFactory,
                 .processor(variantProcessor)
                 .writer(accessionWriter)
                 .faultTolerant()
+                .retry(MissingUnsavedAccessionsException.class)
+                .retryLimit(3)
+                .backOffPolicy(new ExponentialBackOffPolicy())
```

Comment:
Is it worth leaving a comment here or/and in [accession-common](https://github.com/EBIvariation/accession-commons/blob/34a4a2e1daf9227f7a5fcf5bcec2a749cb6161f2/accession-commons-core/src/main/java/uk/ac/ebi/ampt2d/commons/accession/core/BasicAccessioningService.java#L119) to explain the rational of the retry. At least point to the ticket for internal reference.